extensions/blocks/map/edit.js

```diff
@@ -185,6 +212,51 @@ class MapEdit extends Component {
 						] }
 					/>
 					<PanelBody title={ __( 'Map Settings', 'jetpack' ) }>
+						<BaseControl
+							label={ __( 'Height in pixels' ) }
+							id={ `block-jetpack-map-height-input-${ instanceId }` }
+						>
+							<input
+								type="number"
+								id={ `block-jetpack-map-height-input-${ instanceId }` }
+								onChange={ event => {
+									let height = parseInt( event.target.value, 10 );
+									if ( isNaN( height ) ) {
+										// Set map height to default size and input box to empty string
+										height = null;
+									} else if ( height < 400 ) {
+										// Set map height to minimum size
+										height = 400;
+									}
+									setAttributes( {
+										mapHeight: height,
+									} );
+									setTimeout( this.mapRef.current.sizeMap, 0 );
+								} }
+								value={ mapHeight || '' }
+								min="400"
```

Comment:
We are using this value quite a lot across the block.
What about using a constant instead? (also used in my [previous comment](https://github.com/Automattic/jetpack/pull/14706/files?utf8=%E2%9C%93&diff=split&w=1#r380161712) 🙂)
```js
const MIN_HEIGHT = 400;
```

Reply:
👍🏻 f0924df3f